speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
```

Comment:
The invocation name will decide where the stop command goes

"Alexa, stop"

vs

"Alexa, tell Kodi to stop" (though that could be confused with playback